Description: use new macro for initializing libltdl
 Use LTDL_INIT([convenience]) instead of AC_LIBLTDL_CONVENIENCE to fix FTBFS
 while autoreconfing.
Author: Logan Rosen <logan@ubuntu.com>
Last-Update: <2014-01-04>

--- libofa-0.9.3.orig/configure.in
+++ libofa-0.9.3/configure.in
@@ -28,7 +28,7 @@ dnl Checks for programs.
 AC_PREREQ(2.52)
 AC_PROG_AWK
 AC_PROG_CC
-AC_LIBLTDL_CONVENIENCE
+LTDL_INIT([convenience])
 AM_PROG_LIBTOOL
 AC_SUBST(LIBTOOL_DEPS)  
 AC_PROG_CXX
